java/client/src/test/java/glide/api/RedisClientCreateTest.java

```diff
@@ -45,7 +46,12 @@ public void init() {
         connectionManager = mock(ConnectionManager.class);
         threadPoolResource = mock(ThreadPoolResource.class);
 
-        mockedClient.when(() -> buildChannelHandler(any())).thenReturn(channelHandler);
+        RedisClient client = new RedisClient();
```

Comment:
This client needs to close() right? Need a try-with-resources?

Reply:
all instances are mocks here, so I think we can omit that.

Reply:
RedisClient itself isn't a mock. Adding try-with-resources.

Reply:
Nevermind, the client is now mocked.